### PR TITLE
feat:add new files

### PR DESCRIPTION
**Pull request by niraj-24-aptrix**

### CODE REVIEW

File: index.php [hunk 1]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Document</title>
```
Comment (jeetesh-nariya) [inline index.php:6]: please add style css file and then create pull request

File: index.php [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+</head>
+<body>
+    <h1>Hello I am Here</h1>
+
```
Comment (jeetesh-nariya) [inline index.php:10]: also write welcome not hello i am here

**Review by jeetesh-nariya — CHANGES_REQUESTED** [review]
please see all message and re upload code

File: index.php (hunk 1 above)
Comment (niraj-24-aptrix) [reply index.php:6]: ok sir

**Pull request merged**